component/models/dataintegrity/models/models.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright Gen Digital Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package models
+
+import "time"
+
+// TODO integrate VerificationMethod model with the did doc VM model
```

Comment:
In my next PR I'll be taking a look at the TODOs added in this one. If y'all want, I could remove these TODOs for less clutter.

Reply:
I'm neutral on whether the TODOs stay in or not, but thanks for the heads up/explanation!